_includes/team_header.html

```diff
@@ -1,54 +1,68 @@
+{% elsif person.role == "postdoc" or person.role == "phdstudent" or person.role == "phd_rotation" or person.role ==
+"msstudent" or person.role == "undergrad" or person.role =="intern" or person.role =="visitor" or person.role
+=="engineer"%}
+<a class="personKey {{ person.key}}" href="{{ person.website }}">
+  <!--   onclick="selectPerson('{{ person.key }}')">-->
   {% if person.image %}
-    <img alt="{{ person.first_name }} {{ person.last_name }}" class="person-img person-image-small offset{{iter}}" title="{{ person.first_name }} {{ person.last_name }}" src="{{ site.baseurl }}{{ person.image }}" onmouseenter="highlightPerson('{{ person.key }}')" onmouseleave="clearClass('highlighted')">
+  <img alt="{{ person.first_name }} {{ person.last_name }}" class="person-img person-image-small offset{{iter}}"
+    title="{{ person.first_name }} {{ person.last_name }}" src="{{ site.baseurl }}{{ person.image }}"
+    onmouseenter="highlightPerson('{{ person.key }}')" onmouseleave="clearClass('highlighted')">
   {% else %}
-    <img alt="{{ person.first_name }} {{ person.last_name }}" class="person-img person-image-small offset{{iter}}" title="{{ person.first_name }}  {{ person.last_name }}" src="http://www.centonovelosgatos.com/wp-content/uploads/2013/08/unknown-female.jpg" onmouseenter="highlightPerson('{{ person.key }}')" onmouseleave="clearClass('highlighted')">
+  <img alt="{{ person.first_name }} {{ person.last_name }}" class="person-img person-image-small offset{{iter}}"
+    title="{{ person.first_name }}  {{ person.last_name }}"
+    src="http://www.centonovelosgatos.com/wp-content/uploads/2013/08/unknown-female.jpg"
+    onmouseenter="highlightPerson('{{ person.key }}')" onmouseleave="clearClass('highlighted')">
   {% endif %}
-  </a>
-  {% assign iter = iter | plus: 1 | modulo: 2 %}
+</a>
+{% assign iter = iter | plus: 1 | modulo: 2 %}
 {% endif %}
 {% endfor %}
 
 <script>
   // randomness to profile pictures (color, rotation, translation)
   (function () {
-    var people = document.getElementsByClassName("person-img");
-    for(var i in people){
+    const people = document.getElementsByClassName("person-img");
+
+    for (let i in people) {
       // catch non-properties
-      if(!people.hasOwnProperty(i)){
+      if (!people.hasOwnProperty(i)) {
         break;
       }
-      
-      var rTransform = "";
+
+      let rTransform = "";
       // random colors
-      var rColor = Math.floor(Math.random() * 4);
-      people[i].className = people[i].className + " color" + rColor;
-      
+      let rColor = Math.floor(Math.random() * 4);
+      people[i].className = `${people[i].className} color${rColor} ${i > 0 ? 'not-first' : ''}`;
+
       // random rotation
-      var rRotation = Math.floor(Math.random() * 21) - 10;
-      rTransform += "rotate(" + rRotation + "deg)";
-      
+      // const rRotation = Math.floor(Math.random() * 21) - 10;
+      // rTransform += "rotate(" + rRotation + "deg)";
+
```

Comment:
Remove?

Reply:
just in case we want to rotate icons in the future :)

Reply:
Okay, I don't have a strong opinion that it should be removed. We can leave it for now